src/Model/ModelManager.php

```diff
@@ -403,13 +405,14 @@ public function addIdentifiersToQuery(string $class, ProxyQueryInterface $query,
 
     public function batchDelete(string $class, ProxyQueryInterface $query): void
     {
-        $query->select('DISTINCT '.current($query->getRootAliases()));
+        $qb = $query->getQueryBuilder();
```

Comment:
Same for this

Reply:
Check https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1326 :)